src/scheduler.jl

```diff
@@ -125,8 +119,8 @@ function compute_dag(ctx, d::Thunk; options=SchedulerOptions())
                 @warn "Worker $(proc.pid) died on thunk $thunk_id, rescheduling work"
 
                 # Remove dead worker from procs list
+                # Not sure what is desired behaviour if option.singleworker is set...
```

Comment:
We should error out at this point if `single` is set.

Reply:
Ok, will fix!